app-accessibility/flite/flite-2.1.recipe

```diff
@@ -0,0 +1,143 @@
+	cmd:flite_time$commandSuffix = $portVersion
+	lib:libflite$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_grapheme_lang$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_grapheme_lex$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_indic_lang$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_indic_lex$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_time_awb$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_awb$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_kal$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_kal16$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_rms$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_slt$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmulex$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_usenglish$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	flite${secondaryArchSuffix}_devel = $portVersion
+	devel:libflite$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_grapheme_lang$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_grapheme_lex$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_indic_lang$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_indic_lex$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_time_awb$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_awb$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_kal$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_kal16$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_rms$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_slt$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmulex$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_usenglish$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	flite$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage flite$secondaryArchSuffix \
+	$commandBinDir/flite_cmu_time_awb \
+	$commandBinDir/flite_cmu_us_awb \
+	$commandBinDir/flite_cmu_us_kal \
+	$commandBinDir/flite_cmu_us_kal16 \
+	$commandBinDir/flite_cmu_us_rms \
+	$commandBinDir/flite_cmu_us_slt \
+	$commandBinDir/flite_time \
+	$libDir/libflite.so.$libVersion \
+	$libDir/libflite_cmu_indic_lang.so.$libVersion \
+	$libDir/libflite_cmu_indic_lex.so.$libVersion \
+	$libDir/libflite_cmu_time_awb.so.$libVersion \
+	$libDir/libflite_cmu_us_awb.so.$libVersion \
+	$libDir/libflite_cmu_us_kal16.so.$libVersion \
+	$libDir/libflite_cmu_us_slt.so.$libVersion \
+	$libDir/libflite_cmulex.so.$libVersion \
+	$libDir/libflite_usenglish.so.$libVersion
+
+BUILD()
+{
+	autoconf
+	runConfigure --omit-dirs "binDir" ./configure \
+		--bindir="$commandBinDir"
+	make $jobArgs
```

Comment:
$jobArgs was disable in previous version (make # Parallel builds not working), does this work ok now?

Reply:
Seems to be OK, so can stay as is